envs/Harvard-Cannon/config.harvard-cannon.global_inv.yml

```diff
@@ -63,7 +63,7 @@ PointSourceDatasets: ["SRON"]
 ReducedDimensionStateVector: false
 DynamicKFClustering: false
 ClusteringMethod: "kmeans"
-NumberOfElements: 45
+NumberOfElements: 1000
```

Comment:
I think this is greater than the number of native state vector elements in the default configuration, which could be confusing for a user if `ReducedDimensionStateVector` is toggled on in a test run. Switch back to 45

Reply:
The value was raised to 1000 in this config file only, which is meant to serve as a starting point for the global inversion. I selected this value based on approximations from Meghan He's work at 2x2.5. Having only 45 elements in the global inversion doesn't make much sense. 

The value remains 45 in both `config.yml` and `envs/Harvard-Cannon/config.harvard-cannon.yml`.